src/mscorlib/shared/System/Runtime/CompilerServices/RuntimeWrappedException.cs

```diff
@@ -2,41 +2,40 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*=============================================================================
-**
-**
-**
-** Purpose: The exception class uses to wrap all non-CLS compliant exceptions.
-**
-**
-=============================================================================*/
-
 using System;
 using System.Runtime.Serialization;
-using System.Diagnostics.Contracts;
 
 namespace System.Runtime.CompilerServices
 {
+    /// <summary>
+    /// Exception used to wrap all non-CLS compliant exceptions.
+    /// </summary>
     public sealed class RuntimeWrappedException : Exception
     {
-        private RuntimeWrappedException(Object thrownObject)
+        private Object wrappedException; // EE expects this name
+
+#if CORECLR
+        private
+#else
+        // Not an api but has to be public as System.Linq.Expression invokes this through Reflection when an expression
+        // throws an object that doesn't derive from Exception.
+        public
```

Comment:
On a second thought, it may be best to make this public unconditionally and fix Systerm.Linq.Expressions to create it directly, and not using private reflection hack.

Reply:
That will need API review .. I can do that subsequently.

Reply:
API review is required to expose it in contracts. I agree that it would be nice.

System.Linq.Expression can call it without exposing it in contracts. Flipping it to public here does not require API review.